src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMTE_NuclearReactor.java

```diff
@@ -359,6 +362,91 @@ protected GT_OverclockCalculator createOverclockCalculator(@NotNull GT_Recipe re
                 return GT_OverclockCalculator.ofNoOverclock(recipe.mSpecialValue * 4L, recipe.mDuration);
             }
 
+            @NotNull
+            @Override
+            public CheckRecipeResult process() {
+                GT_Recipe_Map recipeMap;
+                if (recipeMapSupplier == null) {
+                    recipeMap = null;
+                } else {
+                    recipeMap = recipeMapSupplier.get();
+                }
+                if (lastRecipeMap != recipeMap) {
+                    lastRecipe = null;
+                    lastRecipeMap = recipeMap;
+                }
+
+                if (maxParallelSupplier != null) {
+                    maxParallel = maxParallelSupplier.get();
+                }
+
+                FindRecipeResult findRecipeResult;
+                if (isRecipeLocked && recipeLockableMachine != null
+                        && recipeLockableMachine.getSingleRecipeCheck() != null) {
+                    // Recipe checker is already built, we'll use it
+                    SingleRecipeCheck singleRecipeCheck = recipeLockableMachine.getSingleRecipeCheck();
+                    // Validate recipe here, otherwise machine will show "not enough output space"
+                    // even if recipe cannot be found
+                    if (singleRecipeCheck.checkRecipeInputs(false, 1, inputItems, inputFluids) == 0) {
+                        return CheckRecipeResultRegistry.NO_RECIPE;
+                    }
+                    findRecipeResult = FindRecipeResult
+                            .ofSuccess(recipeLockableMachine.getSingleRecipeCheck().getRecipe());
+                } else {
+                    findRecipeResult = findRecipe(recipeMap);
+                }
+
+                GT_Recipe recipe;
+                CheckRecipeResult result;
+                if (findRecipeResult.isSuccessful()) {
+                    recipe = findRecipeResult.getRecipeNonNull();
+                    result = validateRecipe(recipe);
+                    if (!result.wasSuccessful()) {
+                        return result;
+                    } else {
+                        lastRecipe = recipe;
+                    }
+                } else {
+                    if (findRecipeResult.getState() == FindRecipeResult.State.INSUFFICIENT_VOLTAGE) {
+                        return CheckRecipeResultRegistry.insufficientPower(findRecipeResult.getRecipeNonNull().mEUt);
+                    } else {
+                        resetMultiProcessing();
```

Comment:
How about this instead of just copying them all?
```java
CheckRecipeResult result = super.process();
if (!result.wasSuccessful()) {
    resetMultiProcessing();
}
return result;
```

Reply:
Good shout, I should have called super in this way.